lib/alephant/broker/request/batch.rb

```diff
@@ -10,17 +10,17 @@ class Batch
         attr_reader :batch_id, :components, :load_strategy
 
         def initialize(component_factory, env)
-          logger.info "Request::Batch#initialize: id: #{env.data['batch_id']}"
+          logger.info "Request::Batch#initialize: id: #{env.data['batch_id']}" if env.data
```

Comment:
Would it be best to do our load test, then work out if we need to remove this based on the results?

This line was previously in here. I just made it less strict, as some of our JS requests are sent without data. Maybe the solution, is to not put these checks in place, but to have a different action for when an OPTIONS request is made. At the moment this goes to our POST endpoint, so I added in these checks.